workshops/checklists/admin.html

```diff
@@ -30,13 +30,12 @@
       This should be done <em>before</em> the workshop is publicly announced.
     </li>
     <li>
-      Either create a 
-      <a href="{{page.root}}/workshops/operations.html#admin_webpage">Github web page</a> for the workshop
-      or ask the lead instructor to do it.
+      Ask the lead instructor to create
+      a <a href="{{page.root}}/workshops/operations.html#admin_webpage">GitHub web page</a>
+      for the workshop.
     </li>
     <li>
-      Add that GitHub repository's URL to <code>config/workshop_urls.yml</code>
-      in the <a href="{{site.github_url}}/site">site</a> repository.
+      Add the URL for that GitHub Pages website to AMY.
```

Comment:
Only the admins have access to AMY (because it contains personal 
identifying information) - we might open it up to instructors in future, 
but for now, the admins are shown how to do this, and they're the only 
ones who can.